src/acceptheaderpayee/handler.js

```diff
@@ -0,0 +1,449 @@
+
+  myCache.set(request.params.id, request.payload)
+
+  histTimerEnd({ success: true, fsp: 'payee', operation: 'putParticipantsByTypeId', source: request.headers['fspiop-source'], destination: request.headers['fspiop-destination'] })
+  return h.response().code(200)
+}
+
+exports.postPartiesByTypeAndId = function (request, h) {
+  const histTimerEnd = Metrics.getHistogram(
+    'sim_request',
+    'Histogram for Simulator http operations',
+    ['success', 'fsp', 'operation', 'source', 'destination']
+  ).startTimer()
+
+  Logger.info('IN PAYEEFSP:: POST /acceptheaderpayeefsp/parties/' + request.params.id, request.payload)
+  myCache.set(request.params.id, request.payload)
+
+  histTimerEnd({ success: true, fsp: 'payee', operation: 'postPartiesByTypeAndId', source: request.headers['fspiop-source'], destination: request.headers['fspiop-destination'] })
+  return h.response().code(202)
+}
+
+exports.getPartiesByTypeAndId = function (req, h) {
+  (async function () {
+    const histTimerEnd = Metrics.getHistogram(
+      'sim_request',
+      'Histogram for Simulator http operations',
+      ['success', 'fsp', 'operation', 'source', 'destination']
+    ).startTimer()
+
+    const metadata = `${req.method} ${req.path} ${req.params.id} `
+    Logger.info((new Date().toISOString()), ['IN PAYEEFSP::'], `received: ${metadata}. `)
+    // Saving Incoming request
+    const incomingRequest = {
+      headers: req.headers
+    }
+    requests.set(req.params.id, incomingRequest)
+
+    const url = partiesEndpoint + `/parties/${req.params.type}/${req.params.id}`
+    try {
+      const protectedHeader = {
+        alg: 'RS256',
+        'FSPIOP-Source': `${req.headers['fspiop-destination']}`,
+        'FSPIOP-Destination': `${req.headers['fspiop-source']}`,
+        'FSPIOP-URI': `/parties/${req.params.type}/${req.params.id}`,
+        'FSPIOP-HTTP-Method': 'PUT',
+        Date: ''
+      }
+      const fspiopSignature = {
+        signature: signature,
+        protectedHeader: `${base64url.encode(JSON.stringify(protectedHeader))}`
+      }
+      const opts = {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/vnd.interoperability.parties+json;version=1.0',
+          'FSPIOP-Source': 'acceptheaderpayeefsp',
+          'FSPIOP-Destination': req.headers['fspiop-source'],
+          Date: new Date().toUTCString(),
+          'FSPIOP-Signature': JSON.stringify(fspiopSignature),
+          'FSPIOP-HTTP-Method': 'PUT',
+          'FSPIOP-URI': `/parties/${req.params.type}/${req.params.id}`,
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(myCache.get(req.params.id))
+      }
+
+      Logger.info((new Date().toISOString()), 'Executing PUT', url)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```